extensions/gcp/processors/PutGCSObject.h

```diff
@@ -58,7 +58,6 @@ class PutGCSObject : public GCSProcessor {
     return utils::array_cat(GCSProcessor::properties(), std::array{
       Bucket,
       Key,
-      NumberOfRetries,
```

Comment:
This is included in [GCSProcessor::properties](https://github.com/apache/nifi-minifi-cpp/blob/main/extensions/gcp/processors/GCSProcessor.h#L46)